native/android/app/build.gradle

```diff
@@ -79,12 +79,12 @@ import com.android.build.OutputFile
 
 project.ext.react = [
         extraPackagerArgs: (System.getenv('EXTRA_PACKAGER_ARGS') ?: "").split("\\s+"),
-        enableHermes: true,  // clean and rebuild if changing
-        entryFile: "index.ts",
+        enableHermes     : true,  // clean and rebuild if changing
```

Comment:
All formatting changes should be reverted in these files.

When we apply the diffs for react native upgrades it is beneficial if they only differ in important parts.